admin.go

```diff
@@ -129,3 +129,45 @@ func (api *Slack) SendSSOBindingEmail(teamName string, user string) error {
 
 	return nil
 }
+
+// SetUltraRestricted converts a user into a single-channel guest
+func (api *Slack) SetUltraRestricted(
+	teamName string,
+	uid string,
+	channel string,
+) error {
```

Comment:
This would probably look better as func(api *Slack) SetUltraRestricted(teamName, uid, channel string) error { instead of the multiline.